docker-bake.hcl

```diff
@@ -414,6 +414,9 @@ target "virtual-osbuild-ci-cXs" {
 
 target "osbuild-ci-c9s-latest" {
         args = {
+                OSB_DNF_PACKAGES = join(",", [
+                        "python3-tomli",
+                ]),
```

Comment:
Won't this effectively override the list defined in `virtual-osbuild-ci-cXs`? :thinking: IOW, don't you need to join this list with it?

Reply:
You are right. This is the reason why `OSB_DNF_PACKAGES` is already copy paste from the base image in the first place.

I was trying to solve this by format new string appending inherited `OSB_DNF_PACKAGES` but no mater I will try - it failed. I think HCL is treating inherited variables some special way so they can be overwritten but cant be used. At least this is my impression after diging documentation for 2h.

The other option would be to add extra `OSB_DNF_UNIQ_PACKAGES` variable and evaluate it down the pipe by dnf scripts. What do you think?

Reply:
Ok, I think I found the right naming schema to access inherited fields from base. Looks nasty but I think it does the trick and only append missing package at the end.